assignments/callbacks.js

```diff
@@ -21,32 +21,110 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
+// Callback #1
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+// ES5
+getLength(items, function(length) {
+  console.log(`There are ${length} items.`);
+});
+
+// Arrow Function
+getLength(items, length => console.log(`There are ${length} items.`));
+
+// Callback #2
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
 
+// ES5
+last(items, function(lastItem) {
+  console.log(lastItem);
+});
+
+// Arrow Function
+last(items, lastItem => console.log(lastItem));
+
+// Callback #3
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
+// ES5
+sumNums(10, 4, function(sum) {
+  console.log(sum);
+});
+
+// Arrow Function
+sumNums(5, 6, sum => console.log(sum));
 
+// Callback #4
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
+//ES5
+multiplyNums(30, 2, function(product) {
+  console.log(product);
+});
+
+// Arrow Function
+multiplyNums(10, 6, product => console.log(product));
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
```

Comment:
this one has a different pattern from the others. Read carefully what it's asking, and compare to the solution code 

```
function contains(item, list, cb) {
  // contains checks if an item is present inside of the given array/list.
  // Pass true to the callback if it is, otherwise pass false.
  for (let i = 0; i < list.length; i++) {
    if (list[i] === item) {
      return cb(true);
    }
  }
  return cb(false);
}
```